cookbooks/resources_allocation.rst

```diff
@@ -0,0 +1,176 @@
+Resources allocation
+====================
+
+SymfonyCloud handles resource allocation in a reserved approach, rather than
+on-demand. This means that your project (applications plus services) are given a
+certain amount of resources and can use as much, or as little, of them as
+needed.
+
+Plan size
+---------
+
+Your production plan size determines how much CPU and RAM are available to your
+entire production environment.
+
+.. note::
+
+   The development plan has no dedicated resources allocated: the main
+   environment is treated like a
+   :ref:`development environment <dev-env-sizing>` and container sizes are set
+   to ``S``.
+
+You can change your project plan at anytime using the
+``symfony project:scale <plan>`` command.
+
+Possible production plan sizes are:
+
+* Standard
+* Medium
+* Large
+* X-Large
+* 2X-Large
+
+Each comes with increasing CPU and memory compared to the previous one. Please
+refer to the `SymfonyCloud pricing page`_ to get accurate resource allocation.
```

Comment:
But this is listed in a section below.

Reply:
bellow is about container size, not plan size.
your comment means we need to clarify this.

what about:
> Each plan comes with increasing CPU and memory compared to the previous one. Please refer to the `SymfonyCloud pricing page`_ to get the official allowance coming with each plan.

?